app/javascript/react/container/ParkShowContainer.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import { Link } from "react-router-dom"
+
+class ParkShowContainer extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      parkShow: {}
+    }
+  }
+
+  render(){
+    return(
+      <div className="park-name">
+        <Link to={`/parks/${this.props.id}`}>{this.props.parkName}</Link>
+      </div>
+    )
+  }
+}
+
+export default ParkShowContainer
+
+
+// {this.props.parkName}
```

Comment:
Same here, and watch empty lines! This file should end at line 21

Reply:
Updated this and removed commented-out code from above note